cpp/test/stats/v_measure.cu

```diff
@@ -93,19 +93,17 @@ class vMeasureTest : public ::testing::TestWithParam<vMeasureParam> {
       truthVMeasure = ((1 + params.beta) * truthHomogeity * truthCompleteness /
                        (params.beta * truthHomogeity + truthCompleteness));
     // calling the v_measure CUDA implementation
-    computedVMeasure = raft::stats::v_measure(truthClusterArray.data(),
-                                              predClusterArray.data(),
-                                              nElements,
-                                              lowerLabelRange,
-                                              upperLabelRange,
-                                              stream,
-                                              params.beta);
+    computedVMeasure = raft::stats::v_measure(
+      handle,
+      raft::make_device_vector_view<const T>(truthClusterArray.data(), nElements),
+      raft::make_device_vector_view<const T>(predClusterArray.data(), nElements),
+      lowerLabelRange,
+      upperLabelRange,
+      params.beta);
   }
 
-  // the destructor
-  void TearDown() override { RAFT_CUDA_TRY(cudaStreamDestroy(stream)); }
-
   // declaring the data values
+  raft::handle_t handle;
```

Comment:
I notice we are missing gtests for a bunch of these. Can you please create an issue and list out the functions which aren't yet being tested? Also, since some of these aren't being tested, can you create a cuml PR and pin it to this branch so we can test that nothing breaks in cuml? We didn't do this for the linalg and matrix APIs and found that some changes actually did break cuml inadvertently.